zerver/openapi/zulip.yaml

```diff
@@ -15327,6 +15354,33 @@ paths:
                           - 2 = Nobody
 
                           **Changes**: New in Zulip 3.0 (feature level 1).
+                      realm_direct_message_initiator_group:
+                        type: integer
+                        allOf:
+                          - description: |
+                              The ID of the [user group](/api/get-user-groups) whose members are
+                              allowed to initiate direct message thread.
+
+                              Previously, access to send direct messages was controlled by the
+                              private_message_policy realm setting, which supported values of
+                              1 (enabled) and 2 (disabled).
+
+                              **Changes**: New in Zulip 9.0 (feature level 262).
+                          - $ref: "#/components/schemas/GroupSettingValue"
+                      realm_direct_message_permission_group:
+                        type: integer
+                        allOf:
+                          - description: |
+                              The ID of the [user group](/api/get-user-groups) of which at least one
+                              member must be included as sender or recipient in all personal and
+                              group direct messages.
```

Comment:
The ID of the user group that has permission to fully use direct messages. Users outside this group can only send direct messages to conversations where all the recipients are in this group, are bots, or are the sender, ensuring that every direct message conversation will be visible to at least one user in this group.